[2025-04-21T17:10:54.172+0000] {processor.py:186} INFO - Started process (PID=671) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:10:54.173+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:10:54.174+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:10:54.173+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:10:54.257+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:10:54.256+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:10:54.257+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:10:54.263+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.093 seconds
[2025-04-21T17:11:24.367+0000] {processor.py:186} INFO - Started process (PID=735) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:11:24.367+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:11:24.369+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:11:24.369+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:11:24.418+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:11:24.417+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:11:24.418+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:11:24.425+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.064 seconds
[2025-04-21T17:11:54.673+0000] {processor.py:186} INFO - Started process (PID=799) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:11:54.674+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:11:54.675+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:11:54.675+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:11:54.726+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:11:54.725+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:11:54.726+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:11:54.733+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T17:12:25.279+0000] {processor.py:186} INFO - Started process (PID=869) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:12:25.280+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:12:25.282+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:12:25.281+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:12:25.340+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:12:25.338+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:12:25.340+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:12:25.350+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.076 seconds
[2025-04-21T17:12:56.054+0000] {processor.py:186} INFO - Started process (PID=933) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:12:56.055+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:12:56.057+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:12:56.057+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:12:56.129+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:12:56.128+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:12:56.130+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:12:56.138+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.089 seconds
[2025-04-21T17:13:26.446+0000] {processor.py:186} INFO - Started process (PID=997) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:13:26.446+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:13:26.448+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:13:26.448+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:13:26.501+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:13:26.500+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:13:26.501+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:13:26.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.069 seconds
[2025-04-21T17:13:56.997+0000] {processor.py:186} INFO - Started process (PID=1061) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:13:56.997+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:13:56.999+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:13:56.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:13:57.048+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:13:57.047+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:13:57.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:13:57.056+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.063 seconds
[2025-04-21T17:14:27.157+0000] {processor.py:186} INFO - Started process (PID=1125) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:14:27.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:14:27.159+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:14:27.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:14:27.208+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:14:27.207+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:14:27.208+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:14:27.215+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.062 seconds
[2025-04-21T17:14:57.576+0000] {processor.py:186} INFO - Started process (PID=1189) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:14:57.577+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:14:57.579+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:14:57.579+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:14:57.627+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:14:57.627+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:14:57.628+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:14:57.635+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.064 seconds
[2025-04-21T17:15:27.709+0000] {processor.py:186} INFO - Started process (PID=1253) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:15:27.710+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:15:27.712+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:15:27.711+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:15:27.771+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:15:27.769+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:15:27.771+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:15:27.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.076 seconds
[2025-04-21T17:15:58.259+0000] {processor.py:186} INFO - Started process (PID=1317) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:15:58.260+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:15:58.262+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:15:58.262+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:15:58.315+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:15:58.314+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:15:58.315+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:15:58.323+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.069 seconds
[2025-04-21T17:16:03.396+0000] {processor.py:186} INFO - Started process (PID=1320) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:16:03.397+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:16:03.399+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:03.399+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:16:03.410+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:03.407+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 125, in <module>
    fetch_book_data_task = PythonOperator(
                           ^^^^^^^^^^^^^^
NameError: name 'PythonOperator' is not defined
[2025-04-21T17:16:03.412+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:16:03.423+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.032 seconds
[2025-04-21T17:16:18.494+0000] {processor.py:186} INFO - Started process (PID=1376) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:16:18.495+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:16:18.496+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.496+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:16:18.504+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:16:18.549+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.549+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:fetch_and_store_amazon_books
[2025-04-21T17:16:18.552+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.552+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:fetch_and_store_amazon_books
[2025-04-21T17:16:18.554+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.554+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:fetch_and_store_amazon_books
[2025-04-21T17:16:18.556+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.556+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:fetch_and_store_amazon_books
[2025-04-21T17:16:18.557+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.557+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:fetch_and_store_amazon_books
[2025-04-21T17:16:18.558+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.558+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:fetch_and_store_amazon_books
[2025-04-21T17:16:18.560+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.560+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:fetch_and_store_amazon_books
[2025-04-21T17:16:18.560+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.560+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T17:16:18.564+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.564+0000] {dag.py:3262} INFO - Creating ORM DAG for fetch_and_store_amazon_books
[2025-04-21T17:16:18.567+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:18.567+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to None, run_after=None
[2025-04-21T17:16:18.572+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.084 seconds
[2025-04-21T17:16:24.561+0000] {processor.py:186} INFO - Started process (PID=1377) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:16:24.562+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:16:24.563+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:24.563+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:16:24.572+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:16:24.581+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:24.580+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T17:16:24.593+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:24.592+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to None, run_after=None
[2025-04-21T17:16:24.600+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.046 seconds
[2025-04-21T17:16:54.788+0000] {processor.py:186} INFO - Started process (PID=1441) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:16:54.788+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:16:54.790+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:54.790+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:16:54.797+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:16:54.811+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:54.810+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T17:16:54.821+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:16:54.821+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to None, run_after=None
[2025-04-21T17:16:54.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.046 seconds
[2025-04-21T17:17:24.950+0000] {processor.py:186} INFO - Started process (PID=1505) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:17:24.950+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:17:24.952+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:17:24.951+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:17:24.955+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:17:24.962+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:17:24.962+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T17:17:24.970+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:17:24.970+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to None, run_after=None
[2025-04-21T17:17:24.976+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.029 seconds
[2025-04-21T17:17:54.414+0000] {processor.py:186} INFO - Started process (PID=1569) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:17:54.415+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:17:54.417+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:17:54.416+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:17:54.467+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:17:54.466+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:17:54.467+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:17:54.474+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.066 seconds
[2025-04-21T17:18:24.604+0000] {processor.py:186} INFO - Started process (PID=1632) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:18:24.605+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:18:24.607+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:18:24.607+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:18:24.659+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:18:24.659+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:18:24.660+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:18:24.669+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.070 seconds
[2025-04-21T17:18:54.999+0000] {processor.py:186} INFO - Started process (PID=1696) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:18:55.000+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:18:55.002+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:18:55.001+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:18:55.052+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:18:55.051+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:18:55.052+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:18:55.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.066 seconds
[2025-04-21T17:19:25.406+0000] {processor.py:186} INFO - Started process (PID=1760) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:19:25.407+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:19:25.407+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:19:25.407+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:19:25.449+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:19:25.448+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:19:25.449+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:19:25.458+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.054 seconds
[2025-04-21T17:19:55.723+0000] {processor.py:186} INFO - Started process (PID=1824) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:19:55.724+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:19:55.724+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:19:55.724+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:19:55.773+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:19:55.772+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:19:55.773+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:19:55.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.061 seconds
[2025-04-21T17:20:26.153+0000] {processor.py:186} INFO - Started process (PID=1893) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:20:26.153+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:20:26.154+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:20:26.154+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:20:26.200+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:20:26.198+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:20:26.200+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:20:26.209+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.059 seconds
[2025-04-21T17:21:13.930+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:21:13.933+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:21:13.936+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:21:13.936+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:21:14.054+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:21:14.051+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:21:14.054+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:21:14.086+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.159 seconds
[2025-04-21T17:21:44.238+0000] {processor.py:186} INFO - Started process (PID=110) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:21:44.239+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:21:44.241+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:21:44.241+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:21:44.297+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:21:44.294+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:21:44.297+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:21:44.308+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.078 seconds
[2025-04-21T17:22:14.506+0000] {processor.py:186} INFO - Started process (PID=174) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:22:14.508+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:22:14.510+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:22:14.510+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:22:14.569+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:22:14.567+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:22:14.569+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:22:14.586+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.085 seconds
[2025-04-21T17:22:44.867+0000] {processor.py:186} INFO - Started process (PID=237) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:22:44.869+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:22:44.871+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:22:44.871+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:22:44.930+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:22:44.928+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:22:44.930+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:22:44.939+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.079 seconds
[2025-04-21T17:23:15.051+0000] {processor.py:186} INFO - Started process (PID=301) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:23:15.052+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:23:15.055+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:23:15.055+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:23:15.104+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:23:15.102+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:23:15.104+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:23:15.114+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.068 seconds
[2025-04-21T17:23:45.185+0000] {processor.py:186} INFO - Started process (PID=365) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:23:45.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:23:45.189+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:23:45.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:23:45.239+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:23:45.238+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:23:45.240+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:23:45.249+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.068 seconds
[2025-04-21T17:24:24.293+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:24:24.293+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:24:24.294+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:24:24.294+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:24:24.393+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:24:24.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:24:24.393+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:24:24.403+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.114 seconds
[2025-04-21T17:24:54.599+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:24:54.601+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:24:54.604+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:24:54.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:24:54.665+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:24:54.664+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:24:54.666+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:24:54.675+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.082 seconds
[2025-04-21T17:25:24.777+0000] {processor.py:186} INFO - Started process (PID=174) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:25:24.778+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:25:24.779+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:25:24.779+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:25:24.830+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:25:24.829+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:25:24.830+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:25:24.836+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.063 seconds
[2025-04-21T17:25:54.879+0000] {processor.py:186} INFO - Started process (PID=238) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:25:54.880+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:25:54.882+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:25:54.882+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:25:54.938+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:25:54.936+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:25:54.939+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:25:54.946+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.070 seconds
[2025-04-21T17:26:25.007+0000] {processor.py:186} INFO - Started process (PID=302) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:26:25.010+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:26:25.011+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:26:25.011+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:26:25.066+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:26:25.065+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:26:25.068+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:26:25.076+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T17:26:55.226+0000] {processor.py:186} INFO - Started process (PID=366) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:26:55.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:26:55.228+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:26:55.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:26:55.280+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:26:55.277+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:26:55.281+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:26:55.289+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.068 seconds
[2025-04-21T17:27:25.378+0000] {processor.py:186} INFO - Started process (PID=430) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:27:25.380+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:27:25.382+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:27:25.382+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:27:25.436+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:27:25.432+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:27:25.436+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:27:25.444+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.073 seconds
[2025-04-21T17:27:55.630+0000] {processor.py:186} INFO - Started process (PID=494) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:27:55.631+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:27:55.632+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:27:55.632+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:27:55.685+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:27:55.683+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:27:55.685+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:27:55.697+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.073 seconds
[2025-04-21T17:28:25.744+0000] {processor.py:186} INFO - Started process (PID=558) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:28:25.746+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:28:25.747+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:28:25.747+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:28:25.806+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:28:25.804+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:28:25.807+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:28:25.817+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.077 seconds
[2025-04-21T17:28:55.905+0000] {processor.py:186} INFO - Started process (PID=621) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:28:55.907+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:28:55.908+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:28:55.908+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:28:55.957+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:28:55.957+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:28:55.958+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:28:55.967+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.066 seconds
[2025-04-21T17:29:26.061+0000] {processor.py:186} INFO - Started process (PID=685) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:29:26.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:29:26.063+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:29:26.063+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:29:26.118+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:29:26.116+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:29:26.118+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:29:26.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.072 seconds
[2025-04-21T17:29:56.672+0000] {processor.py:186} INFO - Started process (PID=749) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:29:56.675+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:29:56.681+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:29:56.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:29:56.742+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:29:56.740+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:29:56.742+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:29:56.752+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.089 seconds
[2025-04-21T17:30:47.265+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:30:47.268+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:30:47.273+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:30:47.273+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:30:47.382+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:30:47.381+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:30:47.384+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:30:47.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.137 seconds
[2025-04-21T17:31:17.583+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:31:17.585+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:31:17.586+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:31:17.586+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:31:17.648+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:31:17.647+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:31:17.651+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:31:17.662+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.085 seconds
[2025-04-21T17:31:47.786+0000] {processor.py:186} INFO - Started process (PID=175) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:31:47.787+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:31:47.789+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:31:47.789+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:31:47.847+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:31:47.843+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:31:47.848+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:31:47.855+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T17:32:17.958+0000] {processor.py:186} INFO - Started process (PID=239) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:32:17.959+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:32:17.962+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:32:17.962+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:32:18.019+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:32:18.018+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:32:18.019+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:32:18.026+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T17:32:48.243+0000] {processor.py:186} INFO - Started process (PID=303) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:32:48.244+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:32:48.246+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:32:48.246+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:32:48.303+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:32:48.302+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:32:48.303+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:32:48.314+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.076 seconds
[2025-04-21T17:33:08.570+0000] {processor.py:186} INFO - Started process (PID=361) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:33:08.571+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:33:08.572+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:33:08.572+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:33:08.625+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:33:08.624+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:33:08.625+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:33:08.633+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.067 seconds
[2025-04-21T17:33:38.842+0000] {processor.py:186} INFO - Started process (PID=425) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:33:38.843+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:33:38.845+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:33:38.844+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:33:38.900+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:33:38.898+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:33:38.900+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:33:38.908+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.072 seconds
[2025-04-21T17:34:09.289+0000] {processor.py:186} INFO - Started process (PID=489) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:34:09.290+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:34:09.292+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:09.292+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:34:09.344+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:09.343+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:34:09.345+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:34:09.353+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.070 seconds
[2025-04-21T17:34:39.722+0000] {processor.py:186} INFO - Started process (PID=553) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:34:39.723+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:34:39.725+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:39.725+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:34:39.776+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:39.775+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:34:39.776+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:34:39.784+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.071 seconds
[2025-04-21T17:34:40.752+0000] {processor.py:186} INFO - Started process (PID=554) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:34:40.752+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:34:40.754+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:40.753+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:34:40.806+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:34:40.805+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:34:40.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:34:40.814+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.067 seconds
[2025-04-21T17:35:34.904+0000] {processor.py:186} INFO - Started process (PID=41) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:35:34.906+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:35:34.907+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:35:34.907+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:35:35.015+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:35:35.014+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:35:35.016+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:35:35.037+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.139 seconds
[2025-04-21T17:36:05.342+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/dag.py
[2025-04-21T17:36:05.343+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T17:36:05.346+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:36:05.346+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T17:36:05.409+0000] {logging_mixin.py:190} INFO - [2025-04-21T17:36:05.407+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T17:36:05.410+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T17:36:05.420+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.085 seconds
[2025-04-21T23:05:27.080+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:05:27.083+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:05:27.089+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:05:27.088+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:05:27.271+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:05:27.268+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:05:27.272+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:05:27.297+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.229 seconds
[2025-04-21T23:05:57.640+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:05:57.641+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:05:57.642+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:05:57.642+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:05:57.700+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:05:57.697+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:05:57.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:05:57.709+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:06:27.753+0000] {processor.py:186} INFO - Started process (PID=168) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:06:27.753+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:06:27.755+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:06:27.755+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:06:27.807+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:06:27.802+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:06:27.808+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:06:27.818+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.068 seconds
[2025-04-21T23:06:57.894+0000] {processor.py:186} INFO - Started process (PID=232) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:06:57.894+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:06:57.895+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:06:57.895+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:06:57.944+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:06:57.941+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:06:57.945+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:06:57.953+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.062 seconds
[2025-04-21T23:07:28.008+0000] {processor.py:186} INFO - Started process (PID=296) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:07:28.008+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:07:28.010+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:07:28.010+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:07:28.062+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:07:28.060+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:07:28.062+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:07:28.072+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.068 seconds
[2025-04-21T23:08:24.886+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:08:24.887+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:08:24.891+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:08:24.890+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:08:25.018+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:08:25.013+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.sql import SQLExecuteQueryOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:08:25.018+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:08:25.034+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.152 seconds
[2025-04-21T23:08:39.914+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:08:39.915+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:08:39.917+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:08:39.917+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:08:39.973+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:08:39.972+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:08:39.973+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:08:39.982+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.072 seconds
[2025-04-21T23:09:10.357+0000] {processor.py:186} INFO - Started process (PID=175) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:09:10.358+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:09:10.360+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:09:10.359+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:09:10.411+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:09:10.410+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:09:10.411+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:09:10.418+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:09:40.573+0000] {processor.py:186} INFO - Started process (PID=239) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:09:40.574+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:09:40.575+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:09:40.575+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:09:40.624+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:09:40.623+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:09:40.624+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:09:40.631+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.062 seconds
[2025-04-21T23:10:10.698+0000] {processor.py:186} INFO - Started process (PID=303) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:10:10.699+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:10:10.701+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:10.701+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:10:10.751+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:10.751+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:10:10.752+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:10:10.759+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:10:40.790+0000] {processor.py:186} INFO - Started process (PID=367) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:10:40.791+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:10:40.795+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:40.795+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:10:40.849+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:40.848+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:10:40.849+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:10:40.856+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.071 seconds
[2025-04-21T23:10:49.906+0000] {processor.py:186} INFO - Started process (PID=368) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:10:49.907+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:10:49.908+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:49.908+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:10:49.966+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:10:49.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:10:49.966+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:10:49.975+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:11:00.124+0000] {processor.py:186} INFO - Started process (PID=378) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:11:00.124+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:11:00.126+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:00.126+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:11:00.183+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:00.182+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:11:00.184+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:11:00.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.072 seconds
[2025-04-21T23:11:19.148+0000] {processor.py:186} INFO - Started process (PID=433) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:11:19.149+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:11:19.151+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:19.150+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:11:19.206+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:19.204+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:11:19.206+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:11:19.214+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.071 seconds
[2025-04-21T23:11:35.174+0000] {processor.py:186} INFO - Started process (PID=497) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:11:35.174+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:11:35.176+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:35.176+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:11:35.227+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:11:35.227+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:11:35.228+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:11:35.234+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:12:06.265+0000] {processor.py:186} INFO - Started process (PID=561) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:12:06.267+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:12:06.269+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:12:06.269+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:12:06.318+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:12:06.318+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:12:06.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:12:06.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.066 seconds
[2025-04-21T23:12:36.699+0000] {processor.py:186} INFO - Started process (PID=625) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:12:36.699+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:12:36.700+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:12:36.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:12:36.738+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:12:36.737+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:12:36.738+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:12:36.745+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.049 seconds
[2025-04-21T23:13:06.795+0000] {processor.py:186} INFO - Started process (PID=689) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:13:06.795+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:13:06.796+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:13:06.796+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:13:06.835+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:13:06.834+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:13:06.835+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:13:06.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.049 seconds
[2025-04-21T23:13:37.247+0000] {processor.py:186} INFO - Started process (PID=728) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:13:37.248+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:13:37.249+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:13:37.249+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:13:37.294+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:13:37.291+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:13:37.294+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:13:37.305+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.060 seconds
[2025-04-21T23:14:07.515+0000] {processor.py:186} INFO - Started process (PID=762) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:14:07.516+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:14:07.517+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:14:07.517+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:14:07.568+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:14:07.567+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:14:07.569+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:14:07.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:14:37.681+0000] {processor.py:186} INFO - Started process (PID=826) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:14:37.681+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:14:37.682+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:14:37.682+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:14:37.721+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:14:37.720+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:14:37.721+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:14:37.728+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.050 seconds
[2025-04-21T23:15:07.815+0000] {processor.py:186} INFO - Started process (PID=890) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:15:07.815+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:15:07.816+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:15:07.816+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:15:07.861+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:15:07.860+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:15:07.861+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:15:07.868+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.057 seconds
[2025-04-21T23:15:37.911+0000] {processor.py:186} INFO - Started process (PID=954) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:15:37.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:15:37.913+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:15:37.913+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:15:37.953+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:15:37.952+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:15:37.954+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:15:37.961+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.052 seconds
[2025-04-21T23:16:07.998+0000] {processor.py:186} INFO - Started process (PID=1018) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:16:07.999+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:16:08.001+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:16:08.000+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:16:08.051+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:16:08.050+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:16:08.051+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:16:08.058+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:17:33.907+0000] {processor.py:186} INFO - Started process (PID=40) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:17:33.908+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:17:33.911+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:17:33.910+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:17:34.011+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:17:34.008+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:17:34.011+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:17:34.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.122 seconds
[2025-04-21T23:18:04.066+0000] {processor.py:186} INFO - Started process (PID=105) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:18:04.070+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:18:04.071+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:18:04.071+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:18:04.123+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:18:04.121+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:18:04.123+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:18:04.132+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.070 seconds
[2025-04-21T23:18:30.203+0000] {processor.py:186} INFO - Started process (PID=167) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:18:30.204+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:18:30.206+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:18:30.206+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:18:30.260+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:18:30.259+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:18:30.261+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:18:30.269+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.071 seconds
[2025-04-21T23:19:00.415+0000] {processor.py:186} INFO - Started process (PID=231) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:19:00.416+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:19:00.418+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:00.417+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:19:00.468+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:00.467+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:19:00.469+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:19:00.476+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.069 seconds
[2025-04-21T23:19:07.563+0000] {processor.py:186} INFO - Started process (PID=241) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:19:07.564+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:19:07.566+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:07.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:19:07.621+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:07.620+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:19:07.622+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:19:07.630+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.070 seconds
[2025-04-21T23:19:37.678+0000] {processor.py:186} INFO - Started process (PID=305) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:19:37.679+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:19:37.680+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:37.680+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:19:37.730+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:19:37.728+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:19:37.730+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:19:37.738+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.065 seconds
[2025-04-21T23:20:07.952+0000] {processor.py:186} INFO - Started process (PID=369) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:20:07.953+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:20:07.954+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:07.954+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:20:08.005+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:08.003+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:20:08.005+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:20:08.012+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.063 seconds
[2025-04-21T23:20:38.441+0000] {processor.py:186} INFO - Started process (PID=433) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:20:38.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:20:38.443+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:38.443+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:20:38.490+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:38.489+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres import PostgresOperator
ImportError: cannot import name 'PostgresOperator' from 'airflow.providers.postgres' (/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/__init__.py)
[2025-04-21T23:20:38.491+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:20:38.500+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.063 seconds
[2025-04-21T23:20:47.451+0000] {processor.py:186} INFO - Started process (PID=488) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:20:47.452+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:20:47.454+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:47.454+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:20:47.864+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:20:47.863+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:20:47.864+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:20:47.872+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.069 seconds
[2025-04-21T23:21:18.122+0000] {processor.py:186} INFO - Started process (PID=552) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:21:18.122+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:21:18.123+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:21:18.123+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:21:18.163+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:21:18.162+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:21:18.164+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:21:18.171+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.051 seconds
[2025-04-21T23:21:48.393+0000] {processor.py:186} INFO - Started process (PID=616) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:21:48.394+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:21:48.396+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:21:48.395+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:21:48.458+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:21:48.457+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:21:48.459+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:21:48.467+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.079 seconds
[2025-04-21T23:22:18.536+0000] {processor.py:186} INFO - Started process (PID=680) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:22:18.537+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:22:18.538+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:22:18.538+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:22:18.580+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:22:18.579+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:22:18.580+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:22:18.587+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.053 seconds
[2025-04-21T23:22:49.003+0000] {processor.py:186} INFO - Started process (PID=744) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:22:49.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:22:49.004+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:22:49.004+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:22:49.048+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:22:49.047+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:22:49.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:22:49.057+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.056 seconds
[2025-04-21T23:23:19.239+0000] {processor.py:186} INFO - Started process (PID=808) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:23:19.240+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:23:19.242+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:19.242+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:23:19.291+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:19.290+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:23:19.291+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:23:19.298+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.063 seconds
[2025-04-21T23:23:45.937+0000] {processor.py:186} INFO - Started process (PID=848) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:23:45.937+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:23:45.938+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:45.938+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:23:45.983+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:23:46.021+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:46.021+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:23:46.028+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:46.028+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2024-06-20 00:00:00+00:00, run_after=2024-06-21 00:00:00+00:00
[2025-04-21T23:23:46.041+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.106 seconds
[2025-04-21T23:23:54.749+0000] {processor.py:186} INFO - Started process (PID=873) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:23:54.750+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:23:54.752+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:54.752+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:23:54.806+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:23:54.811+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:54.811+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:23:54.820+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:23:54.819+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2024-06-20 00:00:00+00:00, run_after=2024-06-21 00:00:00+00:00
[2025-04-21T23:23:54.826+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.082 seconds
[2025-04-21T23:24:24.958+0000] {processor.py:186} INFO - Started process (PID=937) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:24:24.958+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:24:24.960+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:24.960+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:24:25.017+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:24:25.029+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:25.028+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:24:25.041+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:25.041+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2024-06-20 00:00:00+00:00, run_after=2024-06-21 00:00:00+00:00
[2025-04-21T23:24:25.049+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.093 seconds
[2025-04-21T23:24:55.218+0000] {processor.py:186} INFO - Started process (PID=1001) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:24:55.219+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:24:55.221+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:55.221+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:24:55.276+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:24:55.285+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:55.284+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:24:55.293+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:24:55.293+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2024-06-20 00:00:00+00:00, run_after=2024-06-21 00:00:00+00:00
[2025-04-21T23:24:55.299+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.087 seconds
[2025-04-21T23:25:27.474+0000] {processor.py:186} INFO - Started process (PID=1082) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:25:27.477+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:25:27.478+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:25:27.478+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:25:27.529+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:25:27.537+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:25:27.537+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:25:27.552+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.081 seconds
[2025-04-21T23:25:57.708+0000] {processor.py:186} INFO - Started process (PID=1136) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:25:57.709+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:25:57.710+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:25:57.710+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:25:57.753+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:25:57.764+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:25:57.764+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:25:57.782+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.076 seconds
[2025-04-21T23:26:27.894+0000] {processor.py:186} INFO - Started process (PID=1192) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:26:27.895+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:26:27.896+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:26:27.896+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:26:27.939+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:26:27.946+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:26:27.946+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:26:27.966+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:26:58.790+0000] {processor.py:186} INFO - Started process (PID=1252) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:26:58.792+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:26:58.794+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:26:58.794+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:26:58.848+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:26:58.860+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:26:58.860+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:26:58.887+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.102 seconds
[2025-04-21T23:27:29.065+0000] {processor.py:186} INFO - Started process (PID=1316) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:27:29.066+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:27:29.067+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:27:29.067+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:27:29.123+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:27:29.134+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:27:29.134+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:27:29.148+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.089 seconds
[2025-04-21T23:27:59.245+0000] {processor.py:186} INFO - Started process (PID=1380) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:27:59.246+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:27:59.247+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:27:59.247+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:27:59.301+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:27:59.313+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:27:59.313+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:27:59.330+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.090 seconds
[2025-04-21T23:28:29.718+0000] {processor.py:186} INFO - Started process (PID=1444) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:28:29.719+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:28:29.721+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:28:29.721+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:28:29.778+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:28:29.791+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:28:29.791+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:28:29.805+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.092 seconds
[2025-04-21T23:28:59.941+0000] {processor.py:186} INFO - Started process (PID=1508) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:28:59.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:28:59.943+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:28:59.943+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:28:59.997+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:29:00.006+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:29:00.006+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:29:00.024+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.089 seconds
[2025-04-21T23:29:30.191+0000] {processor.py:186} INFO - Started process (PID=1572) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:29:30.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:29:30.194+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:29:30.194+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:29:30.260+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:29:30.274+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:29:30.273+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:29:30.292+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.107 seconds
[2025-04-21T23:29:56.387+0000] {processor.py:186} INFO - Started process (PID=1604) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:29:56.388+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:29:56.390+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:29:56.389+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:29:56.446+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:29:56.458+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:29:56.458+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:29:56.472+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.090 seconds
[2025-04-21T23:30:26.630+0000] {processor.py:186} INFO - Started process (PID=1668) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:30:26.631+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:30:26.632+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:30:26.632+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:30:26.684+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:30:26.695+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:30:26.695+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:30:26.750+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.124 seconds
[2025-04-21T23:30:56.789+0000] {processor.py:186} INFO - Started process (PID=1765) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:30:56.791+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:30:56.793+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:30:56.793+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:30:56.850+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:30:56.858+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:30:56.858+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:30:56.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.090 seconds
[2025-04-21T23:31:27.156+0000] {processor.py:186} INFO - Started process (PID=1828) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:31:27.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:31:27.158+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:31:27.158+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:31:27.209+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:31:27.217+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:31:27.217+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:31:27.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.080 seconds
[2025-04-21T23:31:58.237+0000] {processor.py:186} INFO - Started process (PID=1892) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:31:58.237+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:31:58.239+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:31:58.239+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:31:58.287+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:31:58.294+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:31:58.294+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:31:58.307+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:32:28.395+0000] {processor.py:186} INFO - Started process (PID=1955) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:32:28.396+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:32:28.397+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:32:28.397+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:32:28.445+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:32:28.453+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:32:28.453+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:32:28.465+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:32:59.449+0000] {processor.py:186} INFO - Started process (PID=2019) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:32:59.449+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:32:59.451+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:32:59.451+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:32:59.500+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:32:59.507+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:32:59.507+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:32:59.519+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.075 seconds
[2025-04-21T23:33:30.549+0000] {processor.py:186} INFO - Started process (PID=2083) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:33:30.550+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:33:30.552+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:33:30.551+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:33:30.602+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:33:30.609+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:33:30.609+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:33:30.622+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.078 seconds
[2025-04-21T23:34:00.823+0000] {processor.py:186} INFO - Started process (PID=2147) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:34:00.824+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:34:00.826+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:34:00.826+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:34:00.885+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:34:00.893+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:34:00.893+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:34:00.908+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.091 seconds
[2025-04-21T23:34:31.162+0000] {processor.py:186} INFO - Started process (PID=2211) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:34:31.162+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:34:31.164+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:34:31.164+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:34:31.215+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:34:31.223+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:34:31.223+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:34:31.236+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.080 seconds
[2025-04-21T23:35:01.453+0000] {processor.py:186} INFO - Started process (PID=2275) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:35:01.454+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:35:01.455+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:35:01.455+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:35:01.505+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:35:01.513+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:35:01.513+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:35:01.526+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.077 seconds
[2025-04-21T23:35:31.654+0000] {processor.py:186} INFO - Started process (PID=2339) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:35:31.655+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:35:31.657+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:35:31.656+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:35:31.710+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:35:31.719+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:35:31.718+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:35:31.736+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.087 seconds
[2025-04-21T23:36:02.165+0000] {processor.py:186} INFO - Started process (PID=2446) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:36:02.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:36:02.170+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:36:02.170+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:36:02.240+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:36:02.251+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:36:02.251+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:36:02.272+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.116 seconds
[2025-04-21T23:36:33.180+0000] {processor.py:186} INFO - Started process (PID=2510) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:36:33.181+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:36:33.183+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:36:33.183+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:36:33.254+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:36:33.268+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:36:33.268+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:36:33.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.115 seconds
[2025-04-21T23:37:03.899+0000] {processor.py:186} INFO - Started process (PID=2574) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:37:03.900+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:37:03.901+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:03.901+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:37:03.953+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:37:03.962+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:03.962+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:37:04.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.207 seconds
[2025-04-21T23:37:34.532+0000] {processor.py:186} INFO - Started process (PID=2638) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:37:34.532+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:37:34.534+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:34.534+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:37:34.582+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:37:34.592+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:34.591+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:37:34.608+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.080 seconds
[2025-04-21T23:37:41.099+0000] {processor.py:186} INFO - Started process (PID=2654) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:37:41.101+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:37:41.103+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:41.103+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:37:41.151+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:37:41.147+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:37:41.151+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:37:41.162+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.066 seconds
[2025-04-21T23:38:11.592+0000] {processor.py:186} INFO - Started process (PID=2713) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:38:11.594+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:38:11.595+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:38:11.595+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:38:11.639+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:38:11.637+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:38:11.639+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:38:11.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.057 seconds
[2025-04-21T23:38:41.818+0000] {processor.py:186} INFO - Started process (PID=2772) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:38:41.820+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:38:41.823+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:38:41.823+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:38:41.944+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:38:41.942+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:38:41.945+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:38:41.966+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.155 seconds
[2025-04-21T23:39:12.233+0000] {processor.py:186} INFO - Started process (PID=2836) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:39:12.234+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:39:12.235+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:39:12.235+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:39:12.505+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:39:12.504+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:39:12.505+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:39:12.513+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.285 seconds
[2025-04-21T23:39:43.257+0000] {processor.py:186} INFO - Started process (PID=2900) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:39:43.258+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:39:43.260+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:39:43.260+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:39:43.482+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:39:43.481+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:39:43.482+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:39:43.490+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.239 seconds
[2025-04-21T23:40:13.833+0000] {processor.py:186} INFO - Started process (PID=2964) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:40:13.834+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:40:13.835+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:40:13.835+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:40:14.062+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:40:14.061+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:40:14.062+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:40:14.071+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.242 seconds
[2025-04-21T23:40:44.723+0000] {processor.py:186} INFO - Started process (PID=3028) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:40:44.723+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:40:44.725+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:40:44.725+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:40:44.886+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:40:44.885+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:40:44.886+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:40:44.893+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.174 seconds
[2025-04-21T23:41:15.340+0000] {processor.py:186} INFO - Started process (PID=3092) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:41:15.341+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:41:15.342+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:41:15.341+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:41:15.404+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:41:15.400+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:41:15.405+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:41:15.415+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.079 seconds
[2025-04-21T23:41:38.783+0000] {processor.py:186} INFO - Started process (PID=3156) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:41:38.784+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:41:38.785+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:41:38.784+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:41:38.849+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:41:38.847+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.operators.sql import SQLOperator
ImportError: cannot import name 'SQLOperator' from 'airflow.operators.sql' (unknown location)
[2025-04-21T23:41:38.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:41:38.860+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.083 seconds
[2025-04-21T23:42:09.584+0000] {processor.py:186} INFO - Started process (PID=3220) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:42:09.585+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:42:09.586+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:09.586+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:42:09.793+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:09.792+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:42:09.793+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:42:09.801+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.221 seconds
[2025-04-21T23:42:23.210+0000] {processor.py:186} INFO - Started process (PID=3255) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:42:23.211+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:42:23.216+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:23.215+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:42:23.445+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:42:23.494+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:23.493+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:42:23.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.307 seconds
[2025-04-21T23:42:34.661+0000] {processor.py:186} INFO - Started process (PID=3288) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:42:34.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:42:34.663+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:34.663+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:42:35.016+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:42:35.015+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:42:35.017+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:42:35.029+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.374 seconds
[2025-04-21T23:43:05.638+0000] {processor.py:186} INFO - Started process (PID=3352) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:43:05.638+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:43:05.639+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:43:05.639+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:43:05.815+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:43:05.814+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:43:05.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:43:05.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.189 seconds
[2025-04-21T23:43:36.139+0000] {processor.py:186} INFO - Started process (PID=3416) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:43:36.140+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:43:36.141+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:43:36.140+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:43:36.273+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:43:36.272+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag.py", line 15, in <module>
    from airflow.providers.postgres.operators.postgres import PostgresOperator
ModuleNotFoundError: No module named 'airflow.providers.postgres.operators'
[2025-04-21T23:43:36.273+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:43:36.279+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.146 seconds
[2025-04-21T23:44:04.999+0000] {processor.py:186} INFO - Started process (PID=3480) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:44:05.000+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:44:05.000+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:05.000+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:44:05.170+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:44:05.216+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:05.215+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:44:05.223+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:05.223+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-20 00:00:00+00:00, run_after=2025-04-21 00:00:00+00:00
[2025-04-21T23:44:05.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.233 seconds
[2025-04-21T23:44:35.658+0000] {processor.py:186} INFO - Started process (PID=3544) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:44:35.658+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:44:35.659+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:35.659+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:44:35.804+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:44:35.810+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:35.810+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:44:35.816+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:44:35.816+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:44:35.822+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.169 seconds
[2025-04-21T23:45:05.922+0000] {processor.py:186} INFO - Started process (PID=3608) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:45:05.924+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:45:05.925+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:05.924+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:45:06.069+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:45:06.076+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:06.076+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:45:06.083+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:06.083+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:45:06.088+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.172 seconds
[2025-04-21T23:45:36.812+0000] {processor.py:186} INFO - Started process (PID=3672) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:45:36.812+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:45:36.813+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:36.813+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:45:36.940+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:45:36.947+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:36.947+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:45:36.954+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:45:36.954+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:45:36.960+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.152 seconds
[2025-04-21T23:46:07.459+0000] {processor.py:186} INFO - Started process (PID=3736) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:46:07.460+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:46:07.461+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:07.461+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:46:07.517+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:46:07.527+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:07.526+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:46:07.535+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:07.535+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:46:07.541+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.088 seconds
[2025-04-21T23:46:37.571+0000] {processor.py:186} INFO - Started process (PID=3800) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:46:37.572+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:46:37.573+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:37.572+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:46:37.627+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:46:37.636+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:37.635+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:46:37.644+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:46:37.644+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:46:37.650+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.084 seconds
[2025-04-21T23:47:08.666+0000] {processor.py:186} INFO - Started process (PID=3870) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:47:08.667+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:47:08.667+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:08.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:47:08.715+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:47:08.724+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:08.723+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:47:08.731+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:08.731+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:47:08.737+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.074 seconds
[2025-04-21T23:47:39.679+0000] {processor.py:186} INFO - Started process (PID=3934) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:47:39.680+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:47:39.681+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:39.681+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:47:39.734+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:47:39.742+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:39.742+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:47:39.751+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:47:39.751+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:47:39.757+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.082 seconds
[2025-04-21T23:48:10.710+0000] {processor.py:186} INFO - Started process (PID=3997) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:48:10.711+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:48:10.712+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:10.711+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:48:10.766+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:48:10.775+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:10.774+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:48:10.784+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:10.784+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:48:10.790+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.085 seconds
[2025-04-21T23:48:41.739+0000] {processor.py:186} INFO - Started process (PID=4061) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:48:41.740+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:48:41.741+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:41.741+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:48:41.800+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:48:41.812+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:41.811+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:48:41.825+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:48:41.825+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:48:41.832+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.099 seconds
[2025-04-21T23:49:12.707+0000] {processor.py:186} INFO - Started process (PID=4125) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:49:12.708+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:49:12.709+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:12.709+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:49:12.761+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:49:12.768+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:12.768+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:49:12.776+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:12.776+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:49:12.782+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.080 seconds
[2025-04-21T23:49:42.881+0000] {processor.py:186} INFO - Started process (PID=4189) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:49:42.882+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:49:42.883+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:42.883+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:49:42.933+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:49:42.941+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:42.941+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:49:42.948+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:49:42.948+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:49:42.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.078 seconds
[2025-04-21T23:50:13.874+0000] {processor.py:186} INFO - Started process (PID=4253) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:50:13.875+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:50:13.876+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:13.876+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:50:13.929+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:50:13.937+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:13.936+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:50:13.944+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:13.944+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:50:13.950+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.081 seconds
[2025-04-21T23:50:44.180+0000] {processor.py:186} INFO - Started process (PID=4317) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:50:44.181+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:50:44.182+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:44.182+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:50:44.234+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:50:44.242+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:44.242+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:50:44.249+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:50:44.249+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:50:44.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.080 seconds
[2025-04-21T23:51:14.449+0000] {processor.py:186} INFO - Started process (PID=4381) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:51:14.450+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:51:14.451+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:14.451+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:51:14.510+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:51:14.521+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:14.520+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:51:14.531+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:14.531+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:51:14.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.095 seconds
[2025-04-21T23:51:44.600+0000] {processor.py:186} INFO - Started process (PID=4445) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:51:44.601+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:51:44.602+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:44.602+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:51:44.651+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:51:44.659+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:44.659+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:51:44.667+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:51:44.667+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:51:44.673+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 0.076 seconds
[2025-04-21T23:52:15.686+0000] {processor.py:186} INFO - Started process (PID=4509) to work on /opt/airflow/dags/dag.py
[2025-04-21T23:52:15.687+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag.py for tasks to queue
[2025-04-21T23:52:15.688+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:52:15.688+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag.py
[2025-04-21T23:52:15.739+0000] {processor.py:925} INFO - DAG(s) 'fetch_and_store_amazon_books' retrieved from /opt/airflow/dags/dag.py
[2025-04-21T23:52:17.125+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:52:17.125+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-04-21T23:52:17.157+0000] {logging_mixin.py:190} INFO - [2025-04-21T23:52:17.157+0000] {dag.py:4180} INFO - Setting next_dagrun for fetch_and_store_amazon_books to 2025-04-21 00:00:00+00:00, run_after=2025-04-22 00:00:00+00:00
[2025-04-21T23:52:17.164+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag.py took 1.483 seconds
